docs/build_ios.md

```diff
@@ -0,0 +1,155 @@
+
+<img src="../stores_presence/ios_build/toolchain_env_minus5.gif"/> 
+
+### Background on toolchain/ kivy-ios
+toolchain is both the (1) command and (2) pacakge/suite/collection of software availible in kivy-ios package. toolchain suite of software will will convert the python project (poc-mobile-python) to an Xcode project.
+
+## Step 4: Create Xcode proejct
+Again from inside your KivyBuilds/ do the following command:
+```
+toolchain create openmindset /Users/nick/Documents/poc-mobile-python
+```
+Here openmindset could be anything and the last argument is the path to the Python Kivy project.
+
+## Step 5: Signin to Xcode
+
+Sign in with your [Apple Developer account](https://developer.apple.com/programs/)
+<img src="../stores_presence/ios_build/XcodeSignIn.gif" /> 
+
+## Step 6: Build to simulator or iPhone
+Select device and press the play button
+
+## Build to TestFlight
+1. From inside Xcode select Product menu > Archive
+    - This creates a file that will be submitted to App Store Connects
+2. After Archive (.xcarchive file) is created select the file from Oragnizer and click "Distribute App".
+    - This can also be accessed by Window menu > Organizer
+3. There will be a automated process of verifying the package meets Apple requirements
+    - one issue we have had is a binary that is not accepted from the toolchain environment. The remedy to this is just delete the file and the package will get accepted.
+        - File to delete: KivyBuilds/dist/root/python3/lib/python3.10/site-packages/_watchdow_fsevents.cpython-310-darwin.so
+### Other iOS stuff
+Note:
+- Contributors are welcome to do this one, collected below some beginning instructions with some issues and solutions.
+- Rosetta 2 terminal not required for arm64 machines when using kivy-ios==2.2.1
+
+Good article here:
+- https://nrodrig1.medium.com/put-kivy-application-on-iphone-update-1cda12e79825
+
+Directories structure (according to medium article above)
+```sh
+    _environments/
+        venv_wshKivy/
+    kivyBuilds/
+        build/
+        dist/
+        openmindset-ios/
+    openmindset/
+```
+
+Build notes and frequently used commands:
+
+- we must clone the app from github *WITHOUT* `venv` (wipe the `venv` folder entirely in order to build ios stuff)
+- when doing small code changes within the app, here are the steps to re-create the XCode Project:
+
+```sh
+    rm -rf openmindset-ios
+    toolchain create openmindset /Users/andre.masson/git/perso/python-projects/openmindset
+    cp /Users/andre.masson/git/perso/python-projects/openmindset/data/icon.png openmindset-ios/
+    open openmindset-ios/openmindset.xcodeproj
+```
```

Comment:
You can remove these lines altogether 🔥

Reply:
Yup, agreed. I was going to make that suggestion as well.